index.html

```diff
@@ -0,0 +1,293 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>
+      Digital Credentials API
+    </title>
+    <script src="https://www.w3.org/Tools/respec/respec-w3c" class=
+    "remove"></script>
+    <script class="remove">
+    "use strict";
+    // See https://github.com/w3c/respec/wiki/ for how to configure ReSpec
+    var respecConfig = {
+      github: "WICG/digital-credentials",
+      editors: [
+        {
+          name: "Marcos Caceres",
+          email: "marcosc@apple.com",
+          company: "Apple Inc.",
+          companyURL: "https://apple.com",
+        },
+        // Add additional editors here.
+        // https://github.com/w3c/respec/wiki/editors
+      ],
+      shortName: "digital-credentials",
+      specStatus: "CG-DRAFT",
+      group: "wicg",
+      localBiblio: {
+        "ISO-18013-5": {
+          title: "ISO/IEC 18013-5:2021 Personal identification — ISO-compliant driving licence — Part 5: Mobile driving licence (mDL) application",
+          href: "https://www.iso.org/standard/69084.html",
+          publisher: "ISO",
+          date: "2021-09",
+          status: "Published",
+          seriesInfo: "ISO/IEC 18013-5:2021",
+        },
```

Comment:
```suggestion
      localBiblio: {
        "ISO-18013-5": {
          title: "ISO/IEC 18013-5:2021 Personal identification — ISO-compliant driving licence — Part 5: Mobile driving licence (mDL) application",
          href: "https://www.iso.org/standard/69084.html",
          publisher: "ISO",
          date: "2021-09",
          status: "Published",
          seriesInfo: "ISO/IEC 18013-5:2021",
        },
        "vc-data-model-2.0": {
          title: "The Verifiable Credentials Data Model v2.0",
          href: "https://www.w3.org/TR/vc-data-model-2.0/",
          publisher: "W3C Verifiable Credentials Working Group",
          status: "WD"
        }
```

Reply:
@OR13, vc-data-model-2.0 is already in specref: https://www.specref.org/?q=vc-data-model-2.0

The reason I had to add "ISO-18013-5" to the local biblio is that it's not coming up in specref. I filed a bug to find out why: 
https://github.com/tobie/specref/issues/776

Would you mind dropping this suggestion.

Reply:
Resolving as not needed.